programs/marginfi/src/instructions/marginfi_account/liquidate_start.rs

```diff
@@ -165,3 +213,49 @@ impl Hashable for StartLiquidation<'_> {
         get_discrim_hash("global", "start_liquidation")
     }
 }
+
+// Note: risk admin only
+#[derive(Accounts)]
+pub struct StartDeleverage<'info> {
+    /// Account to deleverage
+    #[account(
+        mut,
+        has_one = liquidation_record,
+        has_one = group,
+        constraint = {
+            let acc = marginfi_account.load()?;
+            !acc.get_flag(ACCOUNT_IN_RECEIVERSHIP)
+                && !acc.get_flag(ACCOUNT_IN_FLASHLOAN)
+                && !acc.get_flag(ACCOUNT_DISABLED)
+        } @MarginfiError::UnexpectedLiquidationState
+    )]
+    pub marginfi_account: AccountLoader<'info, MarginfiAccount>,
+
+    /// The associated liquidation record PDA for the given `marginfi_account`
+    #[account(mut)]
+    pub liquidation_record: AccountLoader<'info, LiquidationRecord>,
+
+    #[account(
+        has_one = risk_admin,
+        constraint = (
+            !group.load()?.is_protocol_paused()
```

Comment:
I suppose we can allow this when paused, since it's admin only. if the admin is pwned they'd probably do the maximum damage nearly instantly anyways